tools/topology/sof/pipe-waves-codec-demux-playback.m4

```diff
@@ -0,0 +1,174 @@
+# Demux Volume Pipeline with Waves codec
+#
+#  Low Latency Playback with demux and volume.
+#
+# Pipeline Endpoints for connection are :-
+#
+#	Playback Demux
+#	B3 (DAI buffer)
+#
+#
+#  host PCM_P -- B0 --> Waves -> B1 --> volume -- B2 --> Demux(M) -- B3 --> sink DAI0
+#                                                              |
+#                                                          pipeline n+1 --> DAI
```

Comment:
what is the demux intended for? 
is this really to go to a DAI, or to create an echo reference?

Reply:
This is from the original. https://github.com/thesofproject/sof/blob/main/tools/topology/sof/pipe-volume-demux-playback.m4 which suggests the comment is wrong in both places.

Reply:
possibly, the topology is riddled with copy/pastes. Isn't that a good opportunity to correct here, and propagate the changes to the original later?

Reply:
I am wondering if its just a loose end  and can be connected to either in reality. In check case it should just be a generic sink

Reply:
as @cujomalainey suggested I copy-pasted from pipe-volume-demux-playback.m4 and add codec adapter to pipe.
How to find out whether demux is needed in the final topology?
Is there a topology-owner person to ask this?

Reply:
Recommend doing a git blame on the original file and then ping them here

Reply:
And yes demux is needed for echo ref path.